dev/io.openliberty.jakarta.mail.2.0_fat/fat/src/io/openliberty/jakarta/mail/fat/FATSuite.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
+ * Copyright (c) 2020 IBM Corporation and others.
```

Comment:
Did you really mean to replace `2017` with `2020`? Shouldn't it be `2017, 2020`?

Reply:
I'll fix that.